src/components/Error404.tsx

```diff
@@ -1,21 +1,28 @@
-import { ReactElement } from 'react';
-import { useNavigate } from 'react-router-dom';
-import ErrorIconSVG from '../assets/errorIcon.svg';
+import { ReactElement } from "react";
+import { useNavigate } from "react-router-dom";
+import ErrorIconSVG from "../assets/errorIcon.svg";
 
-const Error404 = ():ReactElement => {
-    const navigate = useNavigate();
+const Error404 = (): ReactElement => {
+  const navigate = useNavigate();
 
-    return (
-        <div className="flex flex-col items-center justify-center">
-            <img src={ErrorIconSVG} alt="" />
-            <div className="flex flex-col items-center font-sans mt-1 mb-12">
-                <h1 className="text-3xl font-bold text-center">404 - Page Not Found</h1>
-                <p className="text-xl text-center max-w-sm">We can't find the page you're looking for. Contact the site owner if this is a mistake. </p>
-            </div>
-            <button onClick={() => navigate("/events")} className="button-base-red rounded-xl py-3 px-14 md:[py-5 px-20]">Return to home</button>
+  return (
+    <div className="flex flex-col items-center justify-center">
+      <img src={ErrorIconSVG} alt="" />
+      <div className="flex flex-col items-center font-sans mt-1 mb-12">
+        <h1 className="text-3xl font-bold text-center">404 - Page Not Found</h1>
+        <p className="text-xl text-center max-w-sm">
+          We can't find the page you're looking for. Contact the site owner if
+          this is a mistake.{" "}
```

Comment:
If possible I'd remove the {" "} (I'm guessing the auto-format might have done that)